src/include/planner/plannodes/hash_join_plan_node.h

```diff
@@ -54,25 +54,26 @@ class HashJoinPlanNode : public AbstractJoinPlanNode {
       return *this;
     }
 
+    // TODO(WAN) do we want to invalidate the builder after build?
```

Comment:
Wouldn't there be issues from reusing the builder? A lot of the variables are moved from the Builder to the Node, so I don't think you can call build again

Reply:
Yeah. I'm not sure how we'd enforce it right now though.

Reply:
In general it seems ridiculous to have a builder that can only be used once.